topics/sequence-analysis/tutorials/ref-based-rad-seq/tutorial.md

```diff
@@ -39,6 +39,10 @@ You can directly used archive from the Sequence Read Archive (SRA) for raw reads
 
 To download all training datasets (i.e reads, reference genome, population map file and barcodes file), you need to use the corresponding [CeSGO hub](https://cesgo.genouest.org/resources/370/supportingdocs) repository.
```

Comment:
@bernt-matthias is it possible to modify this section to specify the Zenodo repo https://zenodo.org/record/1134547#.WkugUjfjJXI and SRA link or something like that ?